### PR TITLE
Fix Image component

### PR DESCRIPTION
**Pull request by hsiwkin**
[Issue] http://suprem.sec.samsung.net/jira/browse/TIZENWF-2575
[Problem] Image component wasn't working properly
[Solution] I've passed to server info about encoding to make it possible
to save binary files and fixed paths to enable showing saved images
without fulll refresh

Signed-off-by: Hubert Siwkin <h.siwkin@samsung.com>

### CODE REVIEW

File: design-editor/src/panel/property/attribute/attribute-element-image.js [hunk 1]
```diff
@@ -173,11 +173,13 @@ class AttributeImage extends DressElement {
 			const reader = new FileReader();
 			reader.addEventListener('loadend', event => {
 				if (event.target.readyState === FileReader.DONE) {
+					const { basePath } = utils.checkGlobalContext('globalData');
 					const dir = utils.checkGlobalContext('globalData').fileUrl.replace(/[^/]+$/gi, '');
-					const writepath = (`${dir  }/images/${  file.name}`).replace(/\/+/gi, '/');
-					const readpath = (`images/${  file.name}`).replace(/\/+/gi, '/');
 
-					utils.checkGlobalContext('writeFile')(
+					const writepath = path.join(dir, 'images', file.name);
```
Comment (korneliakobiela) [inline design-editor/src/panel/property/attribute/attribute-element-image.js]: In module path-utils, we have the function named `createProjectPath` which make exactly the same as you here. Please use it to follow the DRY rule.

File: design-editor/src/panel/property/attribute/attribute-element-image.js [hunk 2]
```diff
@@ -173,11 +173,13 @@ class AttributeImage extends DressElement {
...
+					const readpath = path.join(basePath, 'images', file.name);
```
Comment (korneliakobiela) [inline design-editor/src/panel/property/attribute/attribute-element-image.js]: Please use a single const to declare all these variables.

File: design-editor/src/vsc/vsc-editor.js [hunk 3]
```diff
@@ -46,163 +45,164 @@ let modelManager = null;
...
+						$('.closet-container').addClass('full design-view-active');
+						self.update(data, window.globalData);
+						element.show();
+						// inform about activate design editor
+						eventEmitter.emit(EVENTS.ActiveEditorUpdated, 1, element);
+					});
+					window.saveToFile = (callback, quiet) => {
+						const modelID = window.globalData.fileUrl;
+						const html = modelManager.getHTML(modelID, true);
+						const save = function () {
```
Comment (korneliakobiela) [inline design-editor/src/vsc/vsc-editor.js]: As above, please use single const

File: design-editor/src/panel/property/attribute/attribute-element-image.js (hunk 1 above)
Comment (hsiwkin) [reply design-editor/src/panel/property/attribute/attribute-element-image.js]: good point - done :)

File: design-editor/src/panel/property/attribute/attribute-element-image.js (hunk 2 above)
Comment (hsiwkin) [reply design-editor/src/panel/property/attribute/attribute-element-image.js]: done

File: design-editor/src/vsc/vsc-editor.js (hunk 3 above)
Comment (hsiwkin) [reply design-editor/src/vsc/vsc-editor.js]: fided

**Pull request merged**